<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notes</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>352</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="customToolbar" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="newNoteButton">
           <property name="text">
            <string>New Note</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 3;
padding: 3;</string>
           </property>
           <property name="placeholderText">
            <string>Search notes, notebooks, and tags</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="userButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="elementary-icons.qrc">
             <normaloff>:/user-info</normaloff>:/user-info</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentArea" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="mainSplitter">
         <property name="styleSheet">
          <string notr="true">QSplitter::handle:horizontal {
    width: 3px;
}

QSplitter::handle:vertical {
    height: 3px;
}</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="opaqueResize">
          <bool>true</bool>
         </property>
         <property name="handleWidth">
          <number>3</number>
         </property>
         <property name="childrenCollapsible">
          <bool>true</bool>
         </property>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth">
           <number>2</number>
          </property>
          <widget class="QTreeWidget" name="notebookTreeWidget">
           <property name="styleSheet">
            <string notr="true">QTreeWidget::item {
    padding: 6 0;
}</string>
           </property>
           <column>
            <property name="text">
             <string>Notebooks</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>Journaling</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Science</string>
            </property>
            <item>
             <property name="text">
              <string>Solar research</string>
             </property>
             <item>
              <property name="text">
               <string>Solar panel pricing over time</string>
              </property>
             </item>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Recipes</string>
            </property>
           </item>
          </widget>
          <widget class="QListWidget" name="testTree">
           <property name="styleSheet">
            <string notr="true">QListWidget::item {
    padding: 6 0;
}</string>
           </property>
           <item>
            <property name="text">
             <string>#app-ideas</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>#recipes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>#summer</string>
            </property>
           </item>
          </widget>
         </widget>
         <widget class="QListWidget" name="noteList">
          <property name="styleSheet">
           <string notr="true">QListWidget::item {
    padding: 6 0;
}</string>
          </property>
          <item>
           <property name="text">
            <string>My First Note</string>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tips for getting the most out of Vibrato</string>
           </property>
          </item>
         </widget>
         <widget class="QFrame" name="noteEditingArea">
          <property name="styleSheet">
           <string notr="true">#noteEditingArea {
	border: 0;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextEdit" name="textEdit">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionPDF"/>
     <addaction name="actionHTML"/>
    </widget>
    <addaction name="actionNew_Note"/>
    <addaction name="actionImport_Notes"/>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionSave"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="actionLog_in"/>
    <addaction name="actionSyncronise"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAccount"/>
  </widget>
  <action name="actionNew_Note">
   <property name="text">
    <string>New Note</string>
   </property>
  </action>
  <action name="actionImport_Notes">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionPDF">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionLog_in">
   <property name="text">
    <string>Log in</string>
   </property>
  </action>
  <action name="actionSyncronise">
   <property name="text">
    <string>Synchronize</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="elementary-icons.qrc"/>
 </resources>
 <connections/>
</ui>
